Makefile

```diff
@@ -72,7 +72,9 @@ verify-nss:
 	$(MAKE) verify -C code/salsa-family
 	$(MAKE) Spec.Chacha20.fst-verify -C specs
 	$(MAKE) ct -C code/poly1305
-	$(MAKE) verify -C code/poly1305
+	# Verification of poly1305 is disabled for now because of CI issues.
```

Comment:
It might be good to slightly change the sentence to make sure that people understand that Poly is forally verified but that verification is disabled in that specific target which is only used by Mozilla's CI